invenio_global_search/serializers/rdm/schema.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 Graz University of Technology.
+#
+# invenio-global-search is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Global Search rdm schema."""
+
+from flask_resources.serializers import BaseSerializerSchema
+from marshmallow import fields
+
+
+class RDMRecordSchema(BaseSerializerSchema):
```

Comment:
In the rdm-records is already a Dublin core serializer. Why do we need an extra serializer?

Reply:
good question!
i knew about the dublin core serializer in rdm-records, but after a closer look i decided not to use it because:
- it serializes communities too
- it references attributes which are not present at the point when invenio-global-search needs the serializer (e.g. DOI)

after thinking it through it could be that i could tackle both reasons with inheriting from the rdm-records  serializer and overriding the not wanted methods. I have to think about that. It could be that the DOI problem is gone because i use now the `publish`method and not more the `create` method

Reply:
Okay, you are correct; these pieces of information are not needed for the global search. It makes sense to create your own serializer.